corehq/apps/accounting/models.py

```diff
@@ -1343,11 +1343,8 @@ def raise_conflicting_dates(self, date_start, date_end):
         ):
             if is_date_range_overlapping(date_start, date_end, sub.date_start, sub.date_end):
                 raise SubscriptionAdjustmentError(
-                    "The start date of %(start_date)s conflicts with the "
-                    "subscription dates to %(related_sub)s." % {
-                        'start_date': self.date_start.strftime(USER_DATE_FORMAT),
-                        'related_sub': sub,
-                    }
+                    "The start date and end date of the new subscription "
+                    f"conflicts with the subscription dates to {sub}."
```

Comment:
nit: if going for accuracy
```suggestion
                    "The start date and/or end date of the new subscription "
                    f"conflicts with the subscription dates to {sub}."
```